bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/PersistentInbox.java

```diff
@@ -97,9 +97,9 @@
      * Internal enumeration to identify the correct type of the event to be fired.
      */
     private enum EventType {
-        added,
-        removed,
-        updated
+    added,
+    removed,
+    updated
```

Comment:
This change was done by the automatic formatting. :cry:

Reply:
yes, very strange behaviour. Reported here: https://bugs.eclipse.org/bugs/show_bug.cgi?id=536322
Please try the workaround and reduce the JavaDoc to a simple comment like `// Internal ....`.

Reply:
Thanks, the workaround seems to work. :+1: